free-courses-vi.md

```diff
@@ -123,6 +123,7 @@
 ### Java
 
 * [Khóa học lập trình JavaFX](https://www.youtube.com/watch?v=zAq7Lmv46PE&list=PL33lvabfss1yRgFCgFXjtYaGAuDJjjH-j) - Kteam
+* [Khóa học lập trình Java cơ bản](https://www.youtube.com/watch?v=mrqVTfpRZCo&list=PLE1qPKuGSJaB4DMiP4wYbLjfszqKg89lL&index=2) - thân triệu
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

```suggestion
* [Học Lập Trình Java Cơ Bản](https://www.youtube.com/playlist?list=PLE1qPKuGSJaB4DMiP4wYbLjfszqKg89lL) - thân triệu
```